flowclient/flowclient/client.py

```diff
@@ -1321,3 +1321,44 @@ def subscriber_degree(
         "direction": direction,
         "subscriber_subset": subscriber_subset,
     }
+
+
+def event_count(
+    *,
+    start: str,
+    stop: str,
+    direction: str = "both",
+    event_types: Union[None, List[str]] = None,
```

Comment:
Can actually write this as `Optional[List[str]]` which is equivalent to `Union[None, ..]` and maybe a bit clearer.